includes/definitions/vertiv-pdu.yaml

```diff
@@ -0,0 +1,16 @@
+os: vertiv-pdu
+text: 'Vertiv PDU'
+type: power
+icon: vertiv
+group: vertiv
+discovery:
+    - sysObjectID:
+        - .1.3.6.1.4.1.21239.42.1.53
+mib_dir: vertiv
+over:
+#   - { graph: vertiv_phase_current, text: Phase Current}
```

Comment:
Please remove the lines if they are not necessary, instead of commenting them.

Reply:
These will be removed.